docs/nb/bgs_main_map.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>Import the &gt; 0.8 filling factor constraint</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahbigwood/S4Mock/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>